pkg/build/builder/common.go

```diff
@@ -130,6 +130,10 @@ func execPostCommitHook(client DockerClient, postCommitSpec api.BuildPostCommitS
 	if err != nil {
 		return fmt.Errorf("read cgroup limits: %v", err)
 	}
+	parent, err := GetCgroupParent()
```

Comment:
that seems like a question for @derekwaynecarr 

if there are systems where we can read the cpu/memory(which we do require today), but can't do this to read the cgroup parent, that's something only he can probably answer.